package.json

```diff
@@ -62,6 +65,7 @@
     "babel-loader": "^6.2.9",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
+    "bluebird": "^3.4.7",
```

Comment:
do we actually need bluebird at this point? I thought `Promise` was supported by every modern browser now (http://caniuse.com/#search=promise)